app/renderer/js/utils/auth-util.ts

```diff
@@ -0,0 +1,39 @@
+import { remote } from 'electron';
+
+import randomstring = require('randomstring');
+import ConfigUtil = require('./config-util');
+
+const { shell } = remote;
+
+class AuthUtil {
+	openInBrowser = (link: string) => {
+		const otp = randomstring.generate({
```

Comment:
We should be sure that we're using a cryptographically secure random string here.  We should look at whether e.g.  https://www.npmjs.com/package/crypto-random-string does anything different from the `randomstring` module.  Though probably in any case, I'd prefer to use something that explicitly documents itself as for cryptographic use.

Reply:
This module does the same thing that ```randomstring``` does. Will still change it.